Release/include/cpprest/details/basic_types.h

```diff
@@ -67,6 +68,7 @@ typedef std::wstringstream stringstream_t;
 typedef char char_t;
 typedef std::string string_t;
 #define _XPLATSTR(x) x
+#define _XPLATTOSTR(x) std::to_string(x)
```

Comment:
Perhaps it makes more sense to define a `template<class T> string_t to_string_t(T&&)`?